core/node/registries/river_registry_contract.go

```diff
@@ -11,6 +11,8 @@ import (
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/gammazero/workerpool"
+	"go.uber.org/atomic"
```

Comment:
Do you need to run `go mod tidy` or something similar to update go.mod file ?

Reply:
Lets use `sync/atomic.Int64`.